### ISSUE TITLE
Update color yellow dark for accessibility

### ISSUE THREAD

**Issue by sarahill** [Shopify/polaris-tokens#42]
# Update color-yellow-dark for accessibility

## Problem

Yellow dark is currently used for text and icons to convey a warning to the merchant in the UI. It doesn't currently pass WCAG constrast standards of 4.5:1.

We started to run into the issue when updating the ExceptionList. See for details https://github.com/Shopify/polaris-react/pull/813

## Solution
I slightly adjusted the brightness to help with contrast. Update `yellow dark` to use `#8A6116` instead of `#9c6f19`.

![image](https://user-images.githubusercontent.com/8629173/50654077-a8cca280-0f59-11e9-96d5-820fccd888ff.png)

---

### Follow these steps

- [x] If you haven’t yet, clone this repository
  ```
  dev clone polaris-tokens
  dev up
  ```
- [x] Fetch tokens from Invision DSM (Design System Manager)
  ```
  dev cd polaris-tokens
  git checkout master
  git pull
  git checkout -B update-color-tokens
  yarn getTokensFromInvisionDSM
  yarn dist
  yarn test -u
  ```
- [x] Update `CHANGELOG.md`
- [x] Commit & push the changes
  ```
  git commit -am "Update color tokens to the latest version"
  git push origin update-color-tokens
  ```
- [x] [Open a pull request][pr-link]

[pr-link]: https://github.com/Shopify/polaris-tokens/compare/master...update-color-tokens?expand=1&title=Update%20color%20tokens&body=Fixes%20%23XXX%20%3C!--%20replace%20XXX%20with%20the%20id%20of%20the%20original%20issue%20--%3E&assignees=kaelig,amrocha,tmlayton

**Comment by sarahill** [comment]
I think I remember how to do it. I will reach out with questions though. Thank you!

**Comment by sarahill** [comment]
I'm having some trouble when I run the following.

After I pull the colors from DSM it seems to be deleting all of the colors except purple. I thought I had done something wrong previously so I closed the branch and started over but same thing. Any ideas @kaelig ?

```
yarn get-tokens-from-invision-dsm
yarn dist
yarn test -u
```

What I'm seeing locally:

![screen shot 2019-01-03 at 5 52 48 pm](https://user-images.githubusercontent.com/8629173/50665973-bc3f3400-0f80-11e9-85ca-dee7ab63fbe0.png)

**Comment by kaelig** [comment]
Sorry about that! It seems like the order in which color palettes come from the API has changed.

I just pushed a fix to `master` in https://github.com/Shopify/polaris-tokens/commit/4ea8f91d9a7b92e24b86ab6a93fbba66256b99e5, can you try again?

**Comment by sarahill** [comment]
That worked! Thanks, Kaelig :) PR opened.

**Comment by kaelig** [comment]
Note: this was released as part of @shopify/polaris-tokens v2.1.0

**Comment by kaelig** [comment]
👋 your project is using an outdated version of polaris-tokens and would benefit from this accessibility fix. Updating to 2.1.1 is warmly recommended 😊 

- [ ] Shopify/online-store-web (cc @Shopify/online-store-web)
- [ ] Shopify/polaris-extended-styleguide (cc @tmlayton)
- [ ] /android (cc @Shopify/mobile-foundations-android)
- [ ] /ios (cc @Shopify/mobile-foundations-ios)

## Upgrade instructions from v1

  - CSS: remove `-base`. For example: `var(--color-ink-base)` → `var(--color-ink)`.
  - Sass: remove `-base`. For example: `$color-ink-base` → `$color-ink`.
  - JSON: remove `-base`. For example: `tokens['color-ink-base']` → `tokens['color-ink']`.
  - Android: remove `_base`. For example: `polaris_color_blue_base` → `polaris_color_blue`.
  - JavaScript: remove `Base`. For example: `colorPurpleBase` → `colorPurple`.

[Full release notes](https://github.com/Shopify/polaris-tokens/blob/master/CHANGELOG.md)
